src/ploneintranet/workspace/browser/tiles/sidebar.py

```diff
@@ -238,12 +236,14 @@ def __call__(self):
         if self.request.method == 'POST' and form:
             ws = self.workspace()
             self.set_grouping_cookie()
-            if not self.can_manage_workspace():
+            wft = api.portal.get_tool("portal_workflow")
+            section = self.request.form.get('section', None)
+            do_reindex = False
+            if section != 'task' and not self.can_manage_workspace():
                 msg = _(u'You do not have permission to change the workspace '
                         u'title or description')
                 raise Unauthorized(msg)
-            do_reindex = False
-            if self.request.form.get('section', None) == 'task':
+            elif section == 'task':
                 current_tasks = self.request.form.get('current-tasks', [])
                 active_tasks = self.request.form.get('active-tasks', [])
```

Comment:
The sidebar tile code is full of far too much logic, and is fast becoming the new `views.py` :(